### PR TITLE
Updated outdated and broken URLs

### PR DESCRIPTION
**Pull request by intgr**
We're in 2018 now and lots has changed...

### CODE REVIEW

**Comment by dvarrazzo** [comment]
Yeah, but I should set up again the website after pgfoundry was taken down..

**Comment by intgr** [comment]
Maybe use https://readthedocs.org/ instead?

**Comment by dvarrazzo** [comment]
Hi @intgr,

thank you very much for the help support. I think this project should receive more help indeed.

My idea is to take care of it and of other few projects in the xmas days, as I have some time to spare. Idea is to merge the outstanding pull requests, clean up the codebase (flake8, black, py2.7+ only), and then publish the docs again. After which just everyday maintenance would do, and help would we welcome.

In case you jump ahead and start merging things or cleaning up things please let me know to avoid duplicating works. Otherwise I will start proceeding one of these CET evenings.

File: pgxnclient/commands/__init__.py [hunk 1]
```diff
@@ -169,7 +169,7 @@ def __make_subparser(self, parser, subparsers,
 
         glb = subp.add_argument_group(_("global options"))
         glb.add_argument("--mirror", metavar="URL",
-            default = 'http://api.pgxn.org/',
+            default = 'https://api.pgxn.org/',
```
Comment (dvarrazzo) [inline pgxnclient/commands/__init__.py]: Note that this has a) changed the behaviour of the program (but it's better this way) and b) broken the test suite. I've fixed it and merged.

Comment (intgr) [reply pgxnclient/commands/__init__.py]: Doh! But this shows how pgxnclient could benefit from CI integration. e.g. #34 :)

**Pull request merged**